src/localization/de/messages.json

```diff
@@ -1,98 +1,98 @@
-  "common.stat.healingPerMana": "",
-  "common.stat.healingPerMana.long": "",
-  "common.stat.healingPerMana.short": "",
-  "common.stat.health": "",
-  "common.stat.intellect": "",
-  "common.stat.leech": "",
-  "common.stat.mana": "",
-  "common.stat.mastery": "",
-  "common.stat.speed": "",
-  "common.stat.stamina": "",
-  "common.stat.strength": "",
-  "common.stat.versatility": "",
-  "common.stat.versatilityDR": "",
-  "common.talents": "",
-  "common.target": "",
-  "common.time": "",
-  "common.total": "",
-  "common.yes": "",
-  "core.abilities.spellCategories.consumable": "",
-  "core.abilities.spellCategories.cooldowns": "",
-  "core.abilities.spellCategories.defensive": "",
-  "core.abilities.spellCategories.healerDamagingSpell": "",
-  "core.abilities.spellCategories.hidden": "",
-  "core.abilities.spellCategories.items": "",
-  "core.abilities.spellCategories.others": "",
-  "core.abilities.spellCategories.rotational": "",
-  "core.abilities.spellCategories.rotationalAoe": "",
-  "core.abilities.spellCategories.semiDefensive": "",
-  "core.abilities.spellCategories.utility": "",
-  "deathknight.blood.suggestions.alwaysBeCasting.downtime": "",
-  "deathknight.blood.suggestions.bloodPlague.uptime": "",
-  "deathknight.blood.suggestions.boneShield.stacksWasted": "",
-  "deathknight.blood.suggestions.boneShield.uptime": "",
-  "deathknight.blood.suggestions.boneStorm.notEnoughTargets": "",
-  "deathknight.blood.suggestions.crimsonScourge.procsWasted": "",
-  "deathknight.blood.suggestions.dancingRuneWeapon.numberCasts": "",
-  "deathknight.blood.suggestions.deathCaress.badCasts": "",
-  "deathknight.blood.suggestions.markOfBlood.uptime": "",
-  "deathknight.blood.suggestions.ossuary.efficiency": "",
-  "deathknight.blood.suggestions.runicPower.wasted": "",
-  "deathknight.blood.suggestions.tomestone.badCasts": "",
-  "deathknight.forst.frostScythe.efficiency": "",
-  "deathknight.frost.suggestions.alwaysBeCasting": "",
-  "deathknight.frost.suggestions.breathOfSindragosa.uptime": "",
-  "deathknight.frost.suggestions.frostFever.uptime": "",
-  "deathknight.frost.suggestions.killingMachine.wasted": "",
-  "deathknight.frost.suggestions.rime.wastedProcs": "",
-  "deathknight.frost.suggestions.runicPower.wasted": "",
-  "deathknight.shared.suggestions.runes.overcapped": "",
-  "deathknight.suggestions.hysteria.efficiency": "",
-  "deathknight.suggestions.swarmingmist.efficiency": "",
-  "deathknight.unholy.suggestions.alwaysBeCasting": "",
-  "deathknight.unholy.suggestions.apocalypse.efficiency": "",
-  "deathknight.unholy.suggestions.festeringStrikes.efficiency": "",
-  "deathknight.unholy.suggestions.runicPower.wasted": "",
-  "deathknight.unholy.suggestions.scourgeStrike.efficiency": "",
-  "deathknight.unholy.suggestions.suddendoom.wastedProcs": "",
-  "deathknight.unholy.suggestions.virulentPlague.uptime": "",
+  "Alle Waffen verbessert (Öle/Steine)": "",
+  "Waffen-Verbesserungen höher Qualität werden verwendet": "",
```

Comment:
On these two, leave the English words on the left and put the translations into the quotes on the right, so it'd look like:
```
"All weapons enhanced (oils/stones)": "Alle Waffen verbessert (Öle/Steine)",
```